### ISSUE TITLE
Why can’t I reproduce the results of the paper?

### ISSUE THREAD

**Issue by languandong** [pengfeiliHEU/M2I2#9]
![Snipaste_2023-11-30_17-15-32](https://github.com/pengfeiliHEU/M2I2/assets/44127403/6c38c906-bcae-4992-9fba-dfd30da759e4)
I reproduced it on the RAD data set following to the instructions you provided.
But the result like above

**Comment by languandong** [comment]
"python3 train_rad.py --checkpoint ./pretrain/med_pretrain_29.pth  --output_dir ./output/rad"

**Comment by pengfeiliHEU** [comment]
The results you reproduced appear relatively normal on the closed set, but they perform poorly on the open set. This is very likely due to how you handled the dataset.

**Comment by Kelvinz-89757** [comment]
@languandong Did you solve the problem of reproducing? I have trouble reproducing the result on RAD dataset too. And it looks like this (freeform only): 
<img width="227" alt="image" src="https://github.com/user-attachments/assets/0da632f2-a46e-4a76-9947-28ed2febe84b">
@pengfeiliHEU  I am figuring what went wrong. Can I ask how you split the data in train and test set? Because I see that in the original evaluation code the phrase type checks "freeform" when use freeform mode. However, in my test data split, the freeform question is marked "test_freeform". This is where it confuses and I modified the evaluation code to be "test_freeform" to fit my test data.

Looking forward to your reply. Thank you very much.
